modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices_{context}"]
+= Adding kernel arguments to enable IOMMU for the BIOS
+
+To enable an IOMMU extension for the BIOS (Basic Input/Output System) associated with a PCI device, create the `MachineConfig` object file and add the kernel arguments for the device.
+
+.Prerequisites
+. Administrative privilege to a working {product-title} cluster.
+. The CPU for the device hardware belongs to either Intel or AMD brand.
```

Comment:
Could you just say, "Intel or AMD CPU hardware" or "Intel or AMD CPU for the device"?

Reply:
Updated the content.